query/src/test/java/org/infinispan/query/distributed/DistributedMassIndexingTest.java

```diff
@@ -69,6 +73,30 @@ public void testReindexing() throws Exception {
       verifyFindsCar(2, "megane");
    }
 
+   public void testPartiallyReindex() throws Exception {
+      loadTestingData();
+      assertEquals(0, getResultListSize("test1", 0));
+      assertEquals(0, getResultListSize("test1", 1));
+      Search.getSearchManager(cache(0)).getMassIndexer().reindex("test1").get();
+      assertEquals(1, getResultListSize("test1", 0));
+      assertEquals(1, getResultListSize("test1", 1));
+   }
+
+   protected int getResultListSize(String keyword, int index) {
+      SearchManager searchManager = Search.getSearchManager(cache(index));
+      Query query = searchManager.buildQueryBuilderForClass(ReindexTestObject.class).get().keyword().onField("name").matching(keyword).createQuery();
```

Comment:
Nothing wrong with using ```ReindexTestObject``` here, but there are 10+ tests that inherit this one that would need to declare this entity as well in the configuration. So I suggest to use the ```Car``` entity which is already configured to facilitate.